packages/expo-updates/e2e/README.md

```diff
@@ -33,7 +33,10 @@ mkdir $WORKING_DIR_ROOT
 - Execute this command to generate the bundles used by the test server:
 
 ```
-yarn generate-test-update-bundles
+// iOS
+yarn generate-test-update-bundles ios
+// android
+yarn generate-test-update-bundles android
```

Comment:
did that break? i remembered that `generate-test-update-bundles` should generate both platforms if not passing the platform argument

Reply:
yeah, it tries to generate web bundle and fails with some error while trying to import from `react-native-web`. Will share the exact error, but for now fixed it in the script [itself](https://github.com/expo/expo/pull/38307/commits/83f7a17b5fd29706b6825b1b8a8d2c5ab64ee6b6)